### PR TITLE
dukto: Add version 6

### LINKED ISSUE #5157: [Request] Add Dukto
Dukto is an easy file transfer tool designed for LAN use...
https://sourceforge.net/projects/dukto/

### PR DESCRIPTION
**Pull request by davidxuang**
- Closes #5157

Provide no `autoupdate` as it is no longer maintained. ([homepage](https://www.msec.it/blog/dukto/))

### CODE REVIEW

File: bucket/dukto.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "version": "6",
+    "description": "An easy file transfer tool designed for LAN use.",
+    "homepage": "https://www.msec.it/blog/dukto/",
+    "license": "GPL-2.0-only",
```
Comment (Ash258) [inline bucket/dukto.json:5]: The license is definitely not GPL

A

```suggestion
    "license": "Freeware",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/dukto.json (hunk 1 above)
Comment (davidxuang) [reply bucket/dukto.json:5]: > The license is definitely not GPL

I'd suggest that it is. (Maybe multi-license?)

image

Comment (davidxuang) [reply bucket/dukto.json:5]: Also on the homepage.

image

Comment (Ash258) [reply bucket/dukto.json:5]: Source code license is irrelevant when there is final product license

Comment (davidxuang) [reply bucket/dukto.json:5]: Weird. Getting confused already.

**Review by Ash258 — APPROVED** [review]

**Pull request merged**